src/main/java/Duke.java

```diff
@@ -1,10 +1,402 @@
+                }
+            }
+        } else {
+            switch (wordArray[0]) {
+                case "todo": {
+                    if (wordArray.length < 2) {
+                        throw new IllegalArgumentException("Blunder! Declare yer to-do as such: 'todo *', ye scurvy dog!");
+                    }
+                    String tempString = input.substring(5).trim();
+                    printDivider();
+                    tasks.add(new Todo(tempString));
+                    taskCount += 1;
+                    System.out.println("I've appended this to yer list:\n" + tasks.get(taskCount - 1).toString());
+                    printDivider();
+                    break;
+                }
+                case "deadline": {
+                    String tempString = input.substring(9).trim();
+                    String[] tempArray = tempString.split("/by", 0);
+                    if (tempArray.length == 1) {
+                        throw new IllegalArgumentException("Blunder! " +
+                                "Declare yer deadline as such: 'deadline * /by *', ye scurvy dog!");
+                    }
+                    String description = tempArray[0].trim();
+                    LocalDateTime by = parseDate(tempArray[1].trim());
+                    printDivider();
+                    tasks.add(new Deadline(description, by));
+                    taskCount += 1;
+                    System.out.println("I've appended this to yer list:\n" + tasks.get(taskCount - 1).toString());
+                    printDivider();
+                    break;
+                }
+                case "event": {
+                    String tempString = input.substring(6).trim();
+                    String[] tempArray = tempString.split("/from", 0);
+                    if (tempArray.length == 1) {
+                        throw new IllegalArgumentException("Blunder! " +
+                                "Declare yer event as such: 'deadline * /from * /to *', ye scurvy dog!");
+                    }
+                    String description = tempArray[0].trim();
+                    tempString = tempArray[1].trim();
+                    tempArray = tempString.split("/to", 0);
+                    if (tempArray.length == 1) {
+                        throw new IllegalArgumentException("Blunder! " +
+                                "Declare yer event as such: 'deadline * /from * /to *', ye scurvy dog!");
+                    }
+                    LocalDateTime from = parseDate(tempArray[0].trim());
+                    LocalDateTime to = parseDate(tempArray[1].trim());
+                    printDivider();
+                    tasks.add(new Event(description, from, to));
+                    taskCount += 1;
+                    System.out.println("I've appended this to yer list:\n" + tasks.get(taskCount - 1).toString());
+                    printDivider();
+                    break;
+                }
+                default:
```

Comment:
Maybe you should try to follow the `switch/case` convention in the [layout guide](https://se-education.org/guides/conventions/java/intermediate.html#layout)? I notice it in other places too.